src/sentry/snuba/metrics_performance.py

```diff
@@ -113,7 +115,6 @@ def bulk_timeseries_query(
     if metrics_compatible:
         with sentry_sdk.start_span(op="mep", description="TimeseriesMetricQueryBuilder"):
             metrics_queries = []
-            metrics_query = None
```

Comment:
The code here is very weird. First, we iterate over `queries`, which means that the last query will be the one setting the value of `metrics_query` which later will have `process_results()` called in line 142.